modules/meteroid/crates/diesel-models/src/query/invoices.rs

```diff
@@ -231,6 +251,33 @@ impl InvoiceRow {
             .into_db_result()
     }
 
+    pub async fn save_invoice_documents(
+        conn: &mut PgConn,
+        id: uuid::Uuid,
+        tenant_id: uuid::Uuid,
+        pdf_document_id: Option<String>,
```

Comment:
this seems to be non-optional in the store, is there a need to have it optional?